docs-source/content/faq/namespace-management.md

```diff
@@ -0,0 +1,137 @@
+---
+title: "Managing Domain Namespaces"
+date: 2019-09-19
+draft: false
+---
+
+Each WebLogic operator deployment manages a number of Kubernetes namespaces (for more information about setting domain namespaces, see [Operator Helm configuration values]({{<relref "/userguide/managing-operators/using-the-operator/using-helm.md#operator-helm-configuration-values">}}). A couple of Kubernetes resources
+have to be present in a namespace before any WebLogic domain custom resources can be successfully 
+deployed into it.
+Those Kubernetes resources are either created as part of the installation
+of the operator's helm chart, or created by the operator at runtime.
+
+This documentation describes a couple of things that you need to be aware when you manage the namespaces while the WebLogic operator is running. For example,
+* Add a namespace for the operator to manage,
+* Delete a namespace from the operator's domain namespace list, or 
+* Delete and recreate a Kubernetes namespace that the WebLogic operator manages.
+
+See more in [Common Mistakes and Solutions]({{<relref "/userguide/managing-operators/using-the-operator/using-helm.md#common-mistakes-and-solutions">}}).
```

Comment:
General comments:
- This is all helpful information IMO.
- Consider asking Rosemary to do an edit pass
- Each of the above three bullets at lines 14, 15, and 16 could be turned into a link to its corresponding section below
- Maybe add a new bullet and corresponding new section that explain how to get the list of the namespaces that a running Operator is already monitoring.  Namely, use `helm ls` to get the list of running operators, and `helm get values my-operator-name` to show the namespaces a particular running operator is monitoring.
- IMO, instead of making it a FAQ entry, this new document looks like it would slot more smoothly into the documentation as a new section in 'using-helm.md' itself (I suspect Rosemary could do this for you if you agree this would be helpful...).

Reply:
Thanks, Tom, for reviewing the doc. Those are all great comments. I'll add links from the introduction area to the individual sections, and add the section for getting the current value of domainNamespaces.
I also feel that the content here fit better in the user guide, instead of the FAQ section.

Updated the doc to address item #3 and #4. 
@rosemarymarano Rosemary, I have done with updating the doc, and you can start working on it anytime.
I'll leave it to Ryan @rjeberhard  and Mark @markxnelson to decide where the content should go.

Reply:
Updated the doc to address item #3 and #4. 
@rosemarymarano Rosemary, I have done with updating the doc, and you can start working on it anytime.
I'll leave it to Ryan @rjeberhard  and Mark @markxnelson to decide where the content should go.

Reply:
Updated the doc to address item #3 and #4. 
@rosemarymarano Rosemary, I have done with updating the doc, and you can start working on it anytime.
I'll leave it to Ryan @rjeberhard  and Mark @markxnelson to decide where the content should go.

Reply:
I am ok either way.